packages/autorest.go/test/databoxedge/armdataboxedge/zz_constants.go

```diff
@@ -303,6 +257,52 @@ func PossibleDayOfWeekValues() []DayOfWeek {
 	}
 }
 
+// DeviceKind - The kind of the device.
+type DeviceKind string
+
+const (
+	DataBoxEdgeDeviceKindAzureDataBoxGateway    DeviceKind = "AzureDataBoxGateway"
```

Comment:
Why isn't this `DeviceKindAzureDataBoxGateway`? The `DataBoxEdge` portion seems unnecessary.

Reply:
Yes good catch, will fix.

Reply:
Fixed